packages/backend/src/App.ts

```diff
@@ -185,7 +196,7 @@ export default class App {
          * request context - for now we simply proxy the existing service repository singleton.
          */
         expressApp.use((req, res, next) => {
-            req.services = serviceRepository;
+            req.services = this.serviceRepository;
```

Comment:
There's probably a good case to instead/also pass the App object itself along the request thread.